<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="rnaseqExamples">
<title>Zhu et al: Effects of SARM1 deficiency on gene expression â€¢ rnaseqExamples</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.1.3/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.1.3/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Zhu et al: Effects of SARM1 deficiency on gene expression">
<meta property="og:description" content="rnaseqExamples">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">rnaseqExamples</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.0.0.9000</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/rnai.html">Naguib et al: Global effects of SUPT4H1 RNAi on gene expression.</a>
    <a class="dropdown-item" href="../articles/sarm1.html">Zhu et al: Effects of SARM1 deficiency on gene expression</a>
    <a class="dropdown-item" href="../articles/tau.html">Wang et al: time course of microglia from the rTg4510 mouse model</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav"></ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Zhu et al: Effects of SARM1 deficiency on gene expression</h1>
            
      
      
      <div class="d-none name"><code>sarm1.Rmd</code></div>
    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/mikelove/DESeq2" class="external-link">DESeq2</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://bioinf.wehi.edu.au/edgeR" class="external-link">edgeR</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tomsing1.github.io/rnaseq-examples/">rnaseqExamples</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">statmod</span><span class="op">)</span>  <span class="co"># required for robust estimates using eBayes()</span></span></code></pre></div>
<p>In 2019, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6446871/" class="external-link">Zhu et al</a> examined gene expression differences between SARM1 deficient and wildtype mouse brains. The authors reported that only a very small number of genes was significantly differentially expressed between the two genotypes in this model, including the <em>Sarm1</em> and its neighboring gene, <em>Xaf1</em>.</p>
<p><strong>Note:</strong> In 2020, <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30387-9" class="external-link">Uccellini et al</a> demonstrated that differential <em>Xaf1</em> expression is <em>not</em> dependent on the presence of <em>Sarm1</em>, but was actually an artifact due to passenger mutations present in the specific Sarm1 knock-out mouse model used by Zhu et al.</p>
<p>A <code>DESeqDataset</code> object with raw counts, sample and gene annotations from the original study by Zhu et al is available in the <code>sarm1</code> dataset included in this R package.</p>
<p>The raw RNA-seq data from this study is available from the <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRP178253" class="external-link">Short Read Archive, accession SRP178253</a> Reads were aligned to the mouse reference genome (version GRCm38_p6, Gencode release release_M17) with the <a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="external-link">STAR aligner (version 2.7.1a)</a> and gene-level abundances were inferred from the BAM files using <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600148/" class="external-link">salmon (version 0.13.1)</a>.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"sarm1"</span><span class="op">)</span></span>
<span><span class="va">dds</span> <span class="op">&lt;-</span> <span class="va">sarm1</span></span></code></pre></div>
<div class="section level2">
<h2 id="differential-expression-analysis-with-deseq2">Differential expression analysis with DESeq2<a class="anchor" aria-label="anchor" href="#differential-expression-analysis-with-deseq2"></a>
</h2>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># filtering, just to speed up computations</span></span>
<span><span class="va">keep</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">rowSums</a></span><span class="op">(</span><span class="fu">counts</span><span class="op">(</span><span class="va">dds</span><span class="op">)</span> <span class="op">&gt;=</span> <span class="fl">10</span><span class="op">)</span> <span class="op">&gt;=</span> <span class="fu"><a href="https://rdrr.io/r/base/Extremes.html" class="external-link">min</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">dds</span><span class="op">$</span><span class="va">group</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">dds</span> <span class="op">&lt;-</span> <span class="va">dds</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/which.html" class="external-link">which</a></span><span class="op">(</span><span class="va">keep</span> <span class="op">&amp;</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">dds</span><span class="op">)</span><span class="op">$</span><span class="va">gene_type</span> <span class="op">==</span> <span class="st">"protein_coding"</span><span class="op">)</span>, <span class="op">]</span></span>
<span><span class="va">dds</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/DESeq2/man/DESeq.html" class="external-link">DESeq</a></span><span class="op">(</span><span class="va">dds</span><span class="op">)</span></span>
<span><span class="co">#&gt; estimating size factors</span></span>
<span><span class="co">#&gt; estimating dispersions</span></span>
<span><span class="co">#&gt; gene-wise dispersion estimates</span></span>
<span><span class="co">#&gt; mean-dispersion relationship</span></span>
<span><span class="co">#&gt; final dispersion estimates</span></span>
<span><span class="co">#&gt; fitting model and testing</span></span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">res</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/DESeq2/man/results.html" class="external-link">results</a></span><span class="op">(</span><span class="va">dds</span>, contrast <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"group"</span>, <span class="st">"KO_ctrl"</span>, <span class="st">"WT_ctrl"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span></span>
<span>  symbol <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">dds</span><span class="op">)</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/row.names.html" class="external-link">row.names</a></span><span class="op">(</span><span class="va">res</span><span class="op">[</span><span class="va">res</span><span class="op">$</span><span class="va">padj</span> <span class="op">&lt;</span> <span class="fl">0.01</span>, <span class="op">]</span><span class="op">)</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"symbol"</span><span class="op">)</span><span class="op">]</span>,</span>
<span>  <span class="va">res</span><span class="op">[</span><span class="va">res</span><span class="op">$</span><span class="va">padj</span> <span class="op">&lt;</span> <span class="fl">0.01</span>, <span class="op">]</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt;                    symbol baseMean log2FoldChange     lfcSE       stat</span></span>
<span><span class="co">#&gt; ENSMUSG00000040483   Xaf1 268.8498       1.144277 0.2224215   5.144634</span></span>
<span><span class="co">#&gt; ENSMUSG00000050132  Sarm1 331.8143      -2.881850 0.1110706 -25.946104</span></span>
<span><span class="co">#&gt;                           pvalue          padj</span></span>
<span><span class="co">#&gt; ENSMUSG00000040483  2.680435e-07  2.015955e-03</span></span>
<span><span class="co">#&gt; ENSMUSG00000050132 2.012012e-148 3.026468e-144</span></span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">DESeq2</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/plotMA.html" class="external-link">plotMA</a></span><span class="op">(</span><span class="va">res</span><span class="op">)</span></span></code></pre></div>
<p><img src="sarm1_files/figure-html/unnamed-chunk-5-1.png" width="576"></p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/DESeq2/man/plotCounts.html" class="external-link">plotCounts</a></span><span class="op">(</span><span class="va">dds</span>, gene <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/which.min.html" class="external-link">which.min</a></span><span class="op">(</span><span class="va">res</span><span class="op">$</span><span class="va">padj</span><span class="op">)</span>, intgroup<span class="op">=</span><span class="st">"group"</span>, </span>
<span>           main <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">dds</span><span class="op">)</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/which.min.html" class="external-link">which.min</a></span><span class="op">(</span><span class="va">res</span><span class="op">$</span><span class="va">padj</span><span class="op">)</span>, <span class="st">"symbol"</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<p><img src="sarm1_files/figure-html/unnamed-chunk-6-1.png" width="576"></p>
</div>
<div class="section level2">
<h2 id="limma-voom-analysis">Limma / Voom analysis<a class="anchor" aria-label="anchor" href="#limma-voom-analysis"></a>
</h2>
<p>Alternatively, we can perform the same analysis using the <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r29" class="external-link">limma/voom</a> framework, specifically the <code><a href="https://rdrr.io/pkg/edgeR/man/voomLmFit.html" class="external-link">voomLmFit()</a></code> function from the <a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html" class="external-link">edgeR</a> Bioconductor package.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">design</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/model.matrix.html" class="external-link">model.matrix</a></span><span class="op">(</span><span class="op">~</span> <span class="fl">0</span> <span class="op">+</span> <span class="va">group</span>, data <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">dds</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">design</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">sub</a></span><span class="op">(</span><span class="st">"group"</span>, <span class="st">""</span>, <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">design</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">keep</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/edgeR/man/filterByExpr.html" class="external-link">filterByExpr</a></span><span class="op">(</span><span class="va">dds</span>, design <span class="op">=</span> <span class="va">design</span><span class="op">)</span></span>
<span><span class="va">fit</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/edgeR/man/voomLmFit.html" class="external-link">voomLmFit</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">dds</span><span class="op">[</span><span class="va">keep</span> <span class="op">&amp;</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">dds</span><span class="op">)</span><span class="op">$</span><span class="va">gene_type</span> <span class="op">==</span> <span class="st">"protein_coding"</span>, <span class="op">]</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  plot <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  sample.weights <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; First sample weights (min/max) 0.468046/1.424113</span></span>
<span><span class="co">#&gt; Final sample weights (min/max) 0.4677988/1.4225225</span></span></code></pre></div>
<p><img src="sarm1_files/figure-html/unnamed-chunk-8-1.png" width="480"></p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="va">contrasts</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/limma/man/makeContrasts.html" class="external-link">makeContrasts</a></span><span class="op">(</span></span>
<span>  SARM1_untreated <span class="op">=</span> <span class="st">"KO_ctrl - WT_ctrl"</span>,</span>
<span>  prion_wt <span class="op">=</span> <span class="st">"WT_prion - WT_ctrl"</span>,</span>
<span>  levels <span class="op">=</span> <span class="va">design</span></span>
<span><span class="op">)</span></span>
<span><span class="va">fit2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/limma/man/contrasts.fit.html" class="external-link">contrasts.fit</a></span><span class="op">(</span><span class="va">fit</span>, contrasts <span class="op">=</span> <span class="va">contrasts</span><span class="op">)</span></span>
<span><span class="va">fit2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/limma/man/ebayes.html" class="external-link">eBayes</a></span><span class="op">(</span><span class="va">fit2</span>, robust <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/limma/man/toptable.html" class="external-link">topTable</a></span><span class="op">(</span><span class="va">fit2</span>, coef <span class="op">=</span> <span class="st">"SARM1_untreated"</span>,</span>
<span>         p.value <span class="op">=</span> <span class="fl">0.05</span><span class="op">)</span><span class="op">[</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"symbol"</span>, <span class="st">"logFC"</span>, <span class="st">"P.Value"</span>, <span class="st">"adj.P.Val"</span><span class="op">)</span><span class="op">]</span></span>
<span><span class="co">#&gt;                    symbol     logFC      P.Value    adj.P.Val</span></span>
<span><span class="co">#&gt; ENSMUSG00000050132  Sarm1 -2.917003 9.224524e-17 1.374546e-12</span></span></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.2.1 (2022-06-23)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 20.04.5 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0</span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       </span></span>
<span><span class="co">#&gt;  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          </span></span>
<span><span class="co">#&gt; [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices datasets  utils     methods  </span></span>
<span><span class="co">#&gt; [8] base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt;  [1] statmod_1.4.37              rnaseqExamples_0.0.0.9000  </span></span>
<span><span class="co">#&gt;  [3] edgeR_3.38.4                limma_3.52.4               </span></span>
<span><span class="co">#&gt;  [5] DESeq2_1.36.0               SummarizedExperiment_1.26.1</span></span>
<span><span class="co">#&gt;  [7] Biobase_2.56.0              MatrixGenerics_1.8.1       </span></span>
<span><span class="co">#&gt;  [9] matrixStats_0.62.0          GenomicRanges_1.48.0       </span></span>
<span><span class="co">#&gt; [11] GenomeInfoDb_1.32.4         IRanges_2.30.1             </span></span>
<span><span class="co">#&gt; [13] S4Vectors_0.34.0            BiocGenerics_0.42.0        </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;  [1] httr_1.4.4             sass_0.4.2             bit64_4.0.5           </span></span>
<span><span class="co">#&gt;  [4] jsonlite_1.8.2         splines_4.2.1          bslib_0.4.0           </span></span>
<span><span class="co">#&gt;  [7] highr_0.9              blob_1.2.3             renv_0.16.0           </span></span>
<span><span class="co">#&gt; [10] GenomeInfoDbData_1.2.8 yaml_2.3.6             pillar_1.8.1          </span></span>
<span><span class="co">#&gt; [13] RSQLite_2.2.18         lattice_0.20-45        glue_1.6.2            </span></span>
<span><span class="co">#&gt; [16] digest_0.6.30          RColorBrewer_1.1-3     XVector_0.36.0        </span></span>
<span><span class="co">#&gt; [19] colorspace_2.0-3       htmltools_0.5.3        Matrix_1.4-1          </span></span>
<span><span class="co">#&gt; [22] pkgconfig_2.0.3        XML_3.99-0.11          genefilter_1.78.0     </span></span>
<span><span class="co">#&gt; [25] zlibbioc_1.42.0        purrr_0.3.5            xtable_1.8-4          </span></span>
<span><span class="co">#&gt; [28] scales_1.2.1           BiocParallel_1.30.4    tibble_3.1.8          </span></span>
<span><span class="co">#&gt; [31] annotate_1.74.0        KEGGREST_1.36.3        ggplot2_3.3.6         </span></span>
<span><span class="co">#&gt; [34] cachem_1.0.6           cli_3.4.1              survival_3.3-1        </span></span>
<span><span class="co">#&gt; [37] magrittr_2.0.3         crayon_1.5.2           memoise_2.0.1         </span></span>
<span><span class="co">#&gt; [40] evaluate_0.17          fansi_1.0.3            fs_1.5.2              </span></span>
<span><span class="co">#&gt; [43] textshaping_0.3.6      tools_4.2.1            lifecycle_1.0.3       </span></span>
<span><span class="co">#&gt; [46] stringr_1.4.1          locfit_1.5-9.6         munsell_0.5.0         </span></span>
<span><span class="co">#&gt; [49] DelayedArray_0.22.0    AnnotationDbi_1.58.0   Biostrings_2.64.1     </span></span>
<span><span class="co">#&gt; [52] compiler_4.2.1         pkgdown_2.0.6          jquerylib_0.1.4       </span></span>
<span><span class="co">#&gt; [55] systemfonts_1.0.4      rlang_1.0.6            grid_4.2.1            </span></span>
<span><span class="co">#&gt; [58] RCurl_1.98-1.9         bitops_1.0-7           rmarkdown_2.17        </span></span>
<span><span class="co">#&gt; [61] gtable_0.3.1           codetools_0.2-18       DBI_1.1.3             </span></span>
<span><span class="co">#&gt; [64] R6_2.5.1               knitr_1.40             utf8_1.2.2            </span></span>
<span><span class="co">#&gt; [67] fastmap_1.1.0          bit_4.0.4              rprojroot_2.0.3       </span></span>
<span><span class="co">#&gt; [70] ragg_1.2.3             desc_1.4.2             stringi_1.7.8         </span></span>
<span><span class="co">#&gt; [73] parallel_4.2.1         Rcpp_1.0.9             vctrs_0.4.2           </span></span>
<span><span class="co">#&gt; [76] geneplotter_1.74.0     png_0.1-7              xfun_0.34</span></span></code></pre></div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Thomas Sandmann.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
